lib/core/src/sdk.rs

```diff
@@ -1132,42 +1165,71 @@ impl LiquidSdk {
     }
 
     /// Prepares to pay to a Bitcoin address via a chain swap.
-    ///
-    /// # Arguments
-    ///
-    /// * `req` - the [PreparePayOnchainRequest] containing:
-    ///     * `receiver_amount_sat` - the amount in satoshi that will be received
-    ///     * `sat_per_vbyte` - the optional fee rate of the Bitcoin claim transaction. Defaults to the swapper estimated claim fee
```

Comment:
Can you update the docs comment with the `amount: PayOnchainAmount`?

Reply:
Done: 2c3fe0a3527adb3d1d331cd4f5491966fb2b24ee